Modelica/Electrical/Analog/UsersGuide/ReleaseNotes.mo

```diff
@@ -3,7 +3,7 @@ class ReleaseNotes "Release Notes"
   extends Modelica.Icons.ReleaseNotes;
   annotation (preferredView="info",Documentation(info="<html>
 
-<h5>Version 4.0.0, 2020-06-04</h5>
+<h5>Version 4.1.0, 2024-03-15</h5>
```

Comment:
```suggestion
<h5>Version 4.0.0, 2020-06-04</h5>
```
As indicated above.

Reply:
Didn't see your changes here before committing the correction myself. @HansOlsson